src/renderer/services/DepositTransactionService.ts

```diff
@@ -0,0 +1,83 @@
+import {Keypair as KeyPair} from "@chainsafe/bls/lib/keypair";
+import bls from "@chainsafe/bls";
+import {BLSPubkey as BLSPubKey, bytes32, bytes48, bytes96, Hash} from "@chainsafe/eth2.0-types";
+import {createHash} from "crypto";
+import {DepositData} from "@chainsafe/eth2.0-types/lib/misc";
+import BN from "bn.js";
+import {signingRoot, SimpleContainerType} from "@chainsafe/ssz";
+import {BLSDomain} from "@chainsafe/bls/lib/types";
+// import abi from "ethereumjs-abi"
+
+// data type definition for DepositData
+const depositDataType: SimpleContainerType = {
+    fields: [
+        ["pubkey", "bytes48"],
+        ["withdrawalCredentials", "bytes32"],
+        ["amount", "uint64"],
+        ["signature", "bytes96"]
+    ]
+};
+
+// fixed deposit amount
+const depositAmount: BN = new BN(30000000000);
+
+const depositBLSDomain: BLSDomain = Buffer.from(Uint8Array.from([3]));
```

Comment:
Domain is 8 bytes.
You should do something like: `Buffer.concat[new BN(3).toArrayLike(Buffer, "le", 4), Buffer.alloc(4)]`

Originally last 4 bytes should be fork but not sure what should we put here

Reply:
In documentation I only found list of [domain types](https://github.com/ethereum/eth2.0-specs/blob/dev/specs/core/0_beacon-chain.md#domain-types).

Reply:
https://github.com/ethereum/eth2.0-specs/blob/dev/specs/core/0_beacon-chain.md#compute_domain

Fork is just array of 4 zero bytes in this case